src/accelerate/utils/fsdp_utils.py

```diff
@@ -27,6 +27,23 @@
 logger = get_logger(__name__)
 
 
+def enable_fsdp_ram_efficient_loading():
+    """
+    Enables RAM efficient loading of Hugging Face models for FSDP in the environment.
```

Comment:
Nit: "in the environment" might not really be helpful for a user.

Reply:
After having Stas complain so much about magical env variables being set, I'd rather over document it hah